config/config.go

```diff
@@ -4,7 +4,7 @@ package config
 const CurrentAppName = "ipfs-update"
 
 // CurrentVersionNumber is the current application's version literal
-const CurrentVersionNumber = "1.7.1"
+const CurrentVersionNumber = "1.8.0-dev"
```

Comment:
@Stebalien does this `dev` pattern make sense for this binary? I don't have a problem continuing it, but I'd like to better understand what we're getting out of it (e.g. is it better user bug reports, etc.).

Reply:
It's not critical... But in theory, it means we can get better version info.